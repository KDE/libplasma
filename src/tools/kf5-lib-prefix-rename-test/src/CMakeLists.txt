find_package(KF6 REQUIRED KIdleTime ItemModels KWidgetsAddons KWindowSystem KCodecs KArchive KCoreAddons Solid ThreadWeaver KF6GlobalAccel
                          KConfig KAuth KJS KWallet KDBusAddons KI18n KGuiAddons KConfigWidgets
                          KService ItemViews KNotifications KIconThemes KCompletion KJobWidgets Sonnet KTextWidgets XmlGui KCrash
                          KBookmarks KDeclarative KUnitConversion KParts Kross KIO KInit KDocTools KDESu)

find_package(KCoreAddons REQUIRED)

find_package(Solid REQUIRED)

find_package(KDESu REQUIRED NO_MODULE)

find_package(KDeclarative REQUIRED NO_MODULE)

find_package(KDE4Support REQUIRED NO_MODULE)

include(PlasmaMacros)
include(PlasmaMacros.cmake)

target_link_libraries(Foo
                     Qt5::Test
                     KF6::Plasma
                     KF6::KArchive KF6::KCoreAddons
                     KF6::KConfigGui
                     KF6::KI18n
                     KF6::KIOCore
                     KF6::KService
                     KF6::PlasmaQuick
                    )

target_link_libraries(plasma-example-kpart-shell KF6::Plasma KF6::KParts)
