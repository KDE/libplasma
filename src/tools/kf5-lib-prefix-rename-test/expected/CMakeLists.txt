find_package(KF6 REQUIRED IdleTime ItemModels WidgetsAddons WindowSystem Codecs Archive CoreAddons Solid ThreadWeaver GlobalAccel
                          Config Auth JS Wallet DBusAddons I18n GuiAddons ConfigWidgets
                          Service ItemViews Notifications IconThemes Completion JobWidgets Sonnet TextWidgets XmlGui Crash
                          Bookmarks Declarative UnitConversion Parts Kross KIO Init DocTools Su)

find_package(KF6CoreAddons REQUIRED)

find_package(KF6Solid REQUIRED)

find_package(KF6Su REQUIRED NO_MODULE)

find_package(KF6Declarative REQUIRED NO_MODULE)

find_package(KF6KDE4Support REQUIRED NO_MODULE)

include(KF6PlasmaMacros)
include(KF6PlasmaMacros.cmake)

target_link_libraries(Foo
                     Qt5::Test
                     KF6::Plasma
                     KF6::Archive KF6::CoreAddons
                     KF6::ConfigGui
                     KF6::I18n
                     KF6::KIOCore
                     KF6::Service
                     KF6::PlasmaQuick
                    )

target_link_libraries(plasma-example-kpart-shell KF6::Plasma KF6::Parts)
